examples/seismic/viscoacoustic/viscoacoustic_example.py

```diff
@@ -43,13 +43,13 @@ def test_viscoacoustic():
 
 if __name__ == "__main__":
     description = ("Example script for a set of viscoacoustic operators.")
-    args = seismic_args(description)
+    args = seismic_args(description).parse_args()
 
     # Preset parameters
     ndim = args.ndim
     shape = args.shape[:args.ndim]
     spacing = tuple(ndim * [10.0])
-    tn = 750. if ndim < 3 else 1250.
+    tn = args.tn if args.tn > 0 else (750. if ndim < 3 else 1250.)
```

Comment:
I'm writing this here, but it applies to all our examples. It feels like that much of the preprocessing could be moved inside `seismic_args` once and for all, instead of being repeated each time in the individual examples?

Reply:
not necessarily for this PR (would be out of scope admittedly), but you know, just saying..

Reply:
I looked into it, and it's not possible if we return the parser from `seismic_args`. We either return the `args` and put all the possible ones in `seismic_args` ir we return the parser and this can only be done afer `parse_args()` which is done here if we want these different cases.

Reply:
I imagined... perhaps that's one reason to switch to `click` -- in which case, we could provide callbacks to each argument! as I said, just food for thought...